lang/python311/Portfile

```diff
@@ -2,11 +2,13 @@
 
 PortSystem 1.0
 PortGroup select 1.0
+PortGroup clang_dependency 1.0
```

Comment:
This cannot work because python311 needs a C11 compiler, which necessitates a dependency on a clang port on some platforms.